src/data/recording/RecordingRepository.ts

```diff
@@ -0,0 +1,41 @@
+import { IStorageRepository } from "../storage/IStorageRepository";
+
+export const RecordingRepository = {
```

Comment:
What's the reason for this to be an object full of methods? 👀

Reply:
I was thinking of naming it with "Repository" so that devs can differentiate methods that

- retrieve data from storage (i.e. methods in the data layer) from
- other methods (e.g. methods in the view and view model layer)

Open to challenges!!